src/saltext/vmware/states/esxi.py

```diff
@@ -957,13 +965,105 @@ def advanced_config(
     return ret
 
 
+def advanced_configs(
+    configs,
+    name="advanced_configs",
```

Comment:
**suggestion** `name` should be the first parameter for states, since it will always be provided by Salt. In terms of the example, `Remove User` will be the name provided here.